flask_resty/view.py

```diff
@@ -493,7 +493,7 @@ class ModelView(ApiView):
     @settable_property
     def session(self):
         """Convenience property for the current SQLAlchemy session."""
-        return flask.current_app.extensions["sqlalchemy"].db.session
+        return flask.current_app.extensions["sqlalchemy"].session
```

Comment:
suggestion: could be made backwards compatible (support both styles)

Reply:
Don't disagree in principle, but there's a bit more changes in tests, etc which make this a bit annoying. 

Upstream dependencies also are major versions, so I figure we could also bump in sync. 

Lastly I don't think we have much external usage, so I opted to optimize for 4c umbrella org usage.